scripts/packet/main.tf.phase1

```diff
@@ -137,10 +179,21 @@ resource "local_file" "vcsa_template" {
   })
   filename = "./tmp/vcsa.json"
   provisioner "local-exec" {
-    command = "sleep 290; echo five more; sleep 290; TERM=xterm-256color ./tmp/mnt/vcsa-cli-installer/lin64/vcsa-deploy install --accept-eula --acknowledge-ceip --no-ssl-certificate-verification --verbose --skip-ovftool-verification ./tmp/vcsa.json"
+    command = "sleep 290; echo five more; sleep 290; TERM=xterm-256color ${var.VCSA_DEPLOY_PATH} install --accept-eula --acknowledge-ceip --no-ssl-certificate-verification --verbose --skip-ovftool-verification $(pwd)/tmp/vcsa.json"
```

Comment:
I recall the sleep 290; echo five more as unnecessary when i set things up... but my memory is hazy

Reply:
sometimes the server is not immediately available, which is why this long sleep was added.